platforms/documentation/docs/src/docs/userguide/dep-man/declaring-repositories/supported_repository_protocols.adoc

```diff
@@ -51,7 +51,7 @@ These protocols determine how Gradle communicates with the repositories to resol
 | <<#sec:gcs-repositories,Documentation>>
 |===
 
-NOTE: Usernames and passwords should never be stored in plain text in your build files. Instead, store credentials in a local `gradle.properties` file or use an open-source Gradle plugin for encrypting and consuming credentials, such as the link:https://plugins.gradle.org/plugin/nu.studer.credentials[credentials plugin].
+NOTE: Usernames and passwords should never be stored in plain text in your build files. Instead, store credentials in a local `gradle.properties` file, or provide the credentials via environment variables.
```

Comment:
We could add links to the [Seauc Credentials plugin](https://github.com/seu-as-code/seu-as-code.plugins) here, similar to how it's done in publishing_gradle_plugins.adoc.
However, the Seauc Credentials plugin doesn’t appear to be actively maintained, and using environment variables seems to be the more common approach nowadays.

I’m not sure if this is the best way to address the issue, so I’d really appreciate your feedback or suggestions.